package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/UrpShaderImporter.cs

```diff
@@ -0,0 +1,178 @@
+
+
+using UnityEngine;
+
+namespace Unity.Formats.USD
+{
+    public class UrpShaderImporter : ShaderImporterBase
```

Comment:
Would be nice to add `///` doc string to the class with brief description and noting any limitations, maybe adding a link to the URP package docs